homeassistant/helpers/event.py

```diff
@@ -272,7 +272,10 @@ def state_change_listener(event: Event[EventStateChangedData]) -> None:
         return async_track_state_change_event(hass, entity_ids, state_change_listener)
 
     return hass.bus.async_listen(
-        EVENT_STATE_CHANGED, state_change_dispatcher, event_filter=state_change_filter
+        EVENT_STATE_CHANGED,
+        state_change_dispatcher,
+        event_filter=state_change_filter,
+        run_immediately=False,
```

Comment:
google_assistant and alexa currently need a delay here for the test to pass. 
They likely don't in production.  Once the tests are fixed we can remove this